tags/guide/fabricdebug.ytag

```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+Add `-Dfabric.log.level=debug` to the VM arguments when running in a development environment
```

Comment:
The tag name is not the reason why you would enable that VM argument. Some explanation would help.